Microsoft.Azure.CosmosRepository/src/Attributes/PartitionKeyPathAttribute.cs

```diff
@@ -1,34 +1,31 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
 
 namespace Microsoft.Azure.CosmosRepository.Attributes
 {
+    using System;
+
     /// <summary>
-    /// The partition key path attribute exposes the ability to declaratively
-    /// specify an <see cref="Item"/> partition key path. This attribute should be used in
-    /// conjunction with a <see cref="Newtonsoft.Json.JsonPropertyAttribute"/> on the <see cref="Item"/> property
-    /// whose value will act as the partition key. Partition key paths should start with "/",
-    /// for example "/partition". For more information,
-    /// see https://docs.microsoft.com/azure/cosmos-db/partitioning-overview.
+    /// The partition key path attribute exposes the ability to declaratively specify an <see
+    /// cref="Item" /> partition key path. This attribute should be used in conjunction with a <see
+    /// cref="Newtonsoft.Json.JsonPropertyAttribute" /> on the <see cref="Item" /> property whose
+    /// value will act as the partition key. Partition key paths should start with "/", for example
+    /// "/partition". For more information, see https://docs.microsoft.com/azure/cosmos-db/partitioning-overview.
     /// </summary>
-    /// <remarks>
-    /// By default, "/id" is used.
-    /// </remarks>
+    /// <remarks>By default, "/id" is used.</remarks>
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
-    public class PartitionKeyPathAttribute : Attribute
+    public sealed class PartitionKeyPathAttribute : Attribute
```

Comment:
I like this being `sealed` good suggestion.

Reply:
That was a code analysis suggestion from VS. :)